pandas/tests/frame/test_operators.py

```diff
@@ -829,6 +829,28 @@ def test_inplace_ops_identity2(self, op):
         expected = id(df)
         assert id(df) == expected
 
+    def test_inplace_drop_and_add(self):
+        # GH 30484
+        # Get expected df
+        expected = pd.DataFrame({})
+        expected["x1"] = [1, 2, 3, 4, 5]
+        expected["x2"] = [0, 0, 0, 1, 1]
+        expected["target"] = [10, 20, 30, 40, 50]
+        y_expected = expected["target"]
+        expected = expected.drop("target", axis=1, inplace=False)
+        y_expected += np.min(y_expected)
+        # Get tested df
+        df = pd.DataFrame({})
+        df["x1"] = [1, 2, 3, 4, 5]
+        df["x2"] = [0, 0, 0, 1, 1]
+        df["target"] = [10, 20, 30, 40, 50]
```

Comment:
test should go in: pandas/tests/frame//test_axis_select_reindex.py

Reply:
I had some trouble on finding where to put the test. Is there any documentation on that?